src/macros.jl

```diff
@@ -490,14 +536,6 @@ function build_constraint(
     return build_constraint(_error, 1.0func, lb, ub)
 end
 
-function build_constraint(_error::Function, expr, lb, ub)
```

Comment:
Where did this go ? This wasn't used before or it's not used now due to a change you made ?

Reply:
It became `function build_constraint(_error::Function, func, set, args...; kwargs...)`

Reply:
It's not sufficient to expect numbers and ranges anymore.

Reply:
Oscar is right, it was removed in favor of the new fallback since it becomes ambiguous when an extra argument is given.

Reply:
Thanks for the explanation